CHANGELOG.md

```diff
@@ -1,5 +1,219 @@
 # Changelog
 
+## 5.0.0.rc1
+
+Teleport 5.0 is a major release with new features, functionality, and
+bug fixes. Users can review [5.0 closed issues](https://github.com/gravitational/teleport/milestone/39?closed=1)
+on Github for details of all items.
+
+#### New Features
+Teleport 5.0 is a major version release and introduces two distinct features: Teleport Application Access and the Unified Access Plane with significant Kubernetes enhancements like multi-cluster support, backwards compatibility, and more.
```

Comment:
Teleport Application Access and significant Kubenetes Access improvements - multi-cluster support...